utils/node_local_test.py

```diff
@@ -145,6 +146,15 @@ def add(self, line, sev, message, cat=None, mtype=None):
         entry['lineStart'] = line.lineno
         entry['description'] = message
         entry['message'] = line.get_anon_msg()
+        if line.filename == 'src/security/cli_security.c':
+            entry['fingerprint'] = '2DAC808D8D0018ECCBBE168CA02DBC1A'
+        elif line.filename == 'src/rdb/rdb_raft.c':
+            entry['filename'] = '972353F8A37813C69F01058745322AA1'
+        else:
+            if cat == 'Fault injection location':
+                entry['fingerprint'] = '{} {}{}'.format(line.filename, line.get_anon_msg())
```

Comment:
(pylint-line-too-long) Line too long (91/80)
(pylint-too-few-format-args) Not enough arguments for format string